runtime/bcutil/ComparingCursor.cpp

```diff
@@ -52,7 +52,8 @@ ComparingCursor::ComparingCursor(J9JavaVM *javaVM, SRPOffsetTable *srpOffsetTabl
 	_mainHelper(srpOffsetTable, romClass, context),
 	_lineNumberHelper(srpOffsetTable, romClass, context),
 	_varInfoHelper(srpOffsetTable, romClass, context),
-	_isEqual(true)
+	_isEqual(true),
+	_isLambda(isLambda)
```

Comment:
I think a better name for this filed can be something like isCompareLamdaFromSCC. We might have code outside shared classes using ComparingCursor in the future.